frontend/css/map.css

```diff
@@ -0,0 +1,105 @@
+body {
+    margin: 0 0 0 0;
+    padding: 0 0 0 0;
+    overflow-y: hidden;
+}
+/* HEADER BOX */
+header {
+    background-color: rgb(167, 60, 113);
+    filter: drop-shadow(0px 10px 10px rgba(0,0,0,.15));
+    width: 100vw;
+    height: 11vh;
+    position: absolute;
+    margin: 0 0 0 0;
+    z-index: 2;
+}
```

Comment:
after we add the links from the footer to our header we'll want to make this a "flex box" element so that it positions the two links && the title evenly across the header

```css
header {
    background-color: #99ddff; /* i also changed the bg color so b/c it was the same color as the link icons */
    filter: drop-shadow(0px 10px 10px rgba(0,0,0,.15));
    width: 100vw;
    height: 11vh;
    position: absolute;
    margin: 0 0 0 0;
    z-index: 2;
   /* this flex box code will make the link icons && title spread evenly across the header responsively */
    display: flex;
    justify-content: space-between;
    align-items: center;
}
```